app/models/level.rb

```diff
@@ -3,4 +3,9 @@ class Level < ActiveRecord::Base
   # has_many :feelings
 
   validates :amount, presence: true
+
+  def categories(level)
```

Comment:
Unused method argument - `level`. If it's necessary, use `_` or `_level` as an argument name to indicate that it won't be used. You can also write as `categories(*)` if you want the method to accept any arguments but don't care about them.